pkg/portforward/portforward.go

```diff
@@ -0,0 +1,82 @@
+package portforward
```

Comment:
It looks like this is only being used in the tests, so it should be moved under `tests/` instead of being included in the main rancher package.

Reply:
Thanks, I moved the portforward package into `/tests`.

As far as I know, the integration and provisioning test suites are similar in that they create downstream clusters the same way, so it might be worth consolidating them in the future. Unlike the provisioning test suit, however, the integration tests seem to be testing a lot of functionality in the Steve API that is not specific to cluster provisioning. I think it would be a lot of work to consolidate them, but it's definitely something we should think about moving forward.